metrics/web/lib/feature_config/presentation/view_models/debug_menu_feature_config_view_model.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
```suggestion
import 'package:meta/meta.dart';
```
I suppose this is for `required` annotation? Maybe `meta` is better?